backend/app/models/visualization_config.py

```diff
@@ -17,6 +13,8 @@ class VisualizationConfigBase(BaseModel):
     job: Optional[EventListenerJobDB]
     client: Optional[str]
     viz_config_data: dict = {}
+    visualization: str
```

Comment:
Can we rename this to `visulization_id` to make it explicit that it is a reference to an ObjectId? I think that's the common practice. Also should the type be PyObjectId?

Eventaually this should probably be `visualization_data_id` or something similar to be more specific (see slack comment).